libretro/libretro_core_options.h

```diff
@@ -39,18 +39,23 @@ struct retro_core_option_definition option_defs_us[] = {
    {
       "snes9x_2010_overclock",
       "SuperFX Overclock",
-      "Overclock or underclock the SuperFX chip. 10Mhz is stock clockspeed.",
+      "Overclock or underclock the SuperFX chip. This may improve the framerate and playability of games that use SuperFX.",
       {
-         { "Underclock(5MHz)", NULL },
-         { "Underclock(8MHz)", NULL },
-         { "Disabled(10MHz)",  NULL },
-         { "40MHz", NULL },
-         { "60MHz", NULL },
-         { "80MHz", NULL },
-         { "100MHz", NULL },
+         { "40 MHz", NULL },
```

Comment:
In the new core option format you can order things gradually and specify the default value at the end (10 MHz here).
So you can have 5/8/9/10/11/12...
It will look better when entering the core option listing (pushing enter/A button in RA on the core option).

Reply:
I'm not sure I understand. This is already using the new core option format right? The default value is listed at the end too. Or do you mean I should use `{ "30", "30 MHz"}` style instead?
I reordered it like this because I thought it made sense to increase a value by pressing up on the d-pad.

Reply:
I meant growing order is what we usually do.

Reply:
Okay. I've reverted the order back to keep it consistent. 😃